src/vs/workbench/browser/actions/listCommands.ts

```diff
@@ -751,7 +751,8 @@ KeybindingsRegistry.registerCommandAndKeybindingRule({
 
 			if (list.getSelection().length > 0) {
 				list.setSelection([]);
-			} else if (list.getFocus().length > 0) {
+			}
+			if (list.getFocus().length > 0) {
```

Comment:
Why this check?

Reply:
It was already there and I just did not remove it.
I guess all those checks do not really make sense now and should be removed.
We should just `setSelection([])` and `setFocus([])`